week2/2-website/css/style.css

```diff
@@ -17,3 +18,216 @@ body {
  */
 
 
+/*-----General styling */
+*{
+    box-sizing: border-box;
+}
+
+.container {
+    margin: 0 auto;
+    min-width: 37.5rem;
+}
+    
+}
+
+ /*------- Navigation */
+
+ header {
+    display: flex;
+    height: 4.375rem;
+}
+
+nav{
+    display: flex;
+    justify-content: space-around;
+    background-color: #ffffff;
+    width: 100%;
+    margin: auto;
+    height: 4.375rem;
+    align-items: center;
+    position: fixed;  
+}
+
+.logo{
+    align-items: center;
+    margin-left: 6.5rem;
+    
+}
+
+.navlinks{
+    display: inline-flex;
+    list-style: none;
+    width: 30%;
+    justify-content: space-around;
+    align-items: center;
+    margin-top: 1.7rem;
+    margin-bottom: 1.7rem;
+    
+}
+
+a{
+    text-decoration: none;
+    color: #838994;
+    font-size: 1.04rem;
+    
+}
+
+a:hover {
+    color: black;
+}
+
+a.active {
+    font-weight: 500;
+    color:#4c5058;
+}
+
+/*------ Intro styling */
+
+.intro{
+    background-image: url('../img/first-background.jpg');
+    background-position: center bottom;
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 45.938rem;
+    overflow: auto;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+
+.intro h1, .intro h3 {
+    color: #ffffff;
+    font-weight: 300;
+}
+
+.intro h1 {
+    font-size: 3.125rem;
+}
+
+.intro h3 {
+    margin-top: 1rem;
+    font-size: 1.563rem;
+}
+
+button {
+    background-color: #f15a29;
+    color: #ffffff;
+    border-style: none;
+    width: 9.688rem;
+    border-radius: 4px;
+    font-size: 1.125rem;
+    padding: 1rem 1.5rem;
+
+}
+
+button:hover {
+    background-color: #d9400e;
+    cursor: pointer;
+}
+
+/*--------Works styling */
+
+.works {
+    background-color: #ffffff;
+    padding-bottom: 5rem;
+}
+
+.works .container {
+    padding: none;
+    text-align: center;
+    color: #1d1e21;
+    margin-top: 7.875rem;
+    text-align: center;
+}
+
+.works h2 {
+    font-weight: 300;
+    font-size: 2.75rem;
+    margin: 1.625rem 0 3.125rem;
+}
+
+.works .how {
+    display: inline-flex;
+    list-style: none;
+    align-items: center;
+    width: 60%;
```

Comment:
This 60% can make this section a little bit tighter than it needs to be.

By not adding the width, it would take up the same size as it's contents.

Reply:
I have taken it out and left it to what you suggested.